tools/bin/commands/install

```diff
@@ -6,9 +6,12 @@ install::description() {
 
 install::usage() {
     cat <<EOT
--s  --setup-as-admin <user>   Install CRDs clusterwide and optionally add permission to the given user so that
-                              user can install the operator in his projects. Othwerwise only admin can install the
-                              operator. You have to run this once as cluster admin
+-s  --setup-as-admin          Install CRDs clusterwide. Use --prepare-for-user if you want a specific user to be
```

Comment:
would it make sense to call this just `--setup`, `install_syndesis_crd` fails if the user isn't admin and prints a helpful message